Fix the following lintian issues:

W: floppyd: manpage-has-errors-from-man usr/share/man/man1/floppyd.1.gz 23: warning: `iX' not defined
W: floppyd: manpage-has-errors-from-man usr/share/man/man1/floppyd.1.gz 110: warning: `lp' not defined

man --warnings -l mtools.5 >/dev/null
<standard input>:17: warning: `oh' not defined
<standard input>:18: warning: `eh' not defined
<standard input>:34: warning: `iX' not defined
<standard input>:137: warning: `lp' not defined

Patched-by: Anibal Monsalve Salazar <anibal@debian.org>
Signed-off-by: Anibal Monsalve Salazar <anibal@debian.org>

--- a/floppyd.1	2010-02-15 12:37:35.000000000 +1100
+++ b/floppyd.1	2010-02-15 14:01:02.000000000 +1100
@@ -20,9 +20,6 @@ documentation, and may not be entirely a
 end of this man page for details.
 .PP
 .SH Description
-.iX "p floppyd"
-.iX "c X terminal"
-.iX "c remote floppy access"
 .PP
 \&\fR\&\f(CWFloppyd\fR is used as a server to grant access to the floppy drive
 to clients running on a remote machine, just as an X server grants
@@ -110,7 +107,6 @@ floppyd -d /dev/fd0
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
@@ -127,7 +123,6 @@ floppyd -d -x :1 -p 5704 /dev/fd0 
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
@@ -143,7 +138,6 @@ floppyd-1    5704/tcp    # floppy daemon
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
@@ -160,7 +154,6 @@ floppyd-1 stream  tcp  wait  floppy  /us
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
@@ -178,7 +171,6 @@ drive x: file="$DISPLAY" remote
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
@@ -193,7 +185,6 @@ drive z: file="$DISPLAY//2" remote
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
@@ -220,7 +211,6 @@ commands:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .TP
@@ -234,7 +224,6 @@ To generate a html copy,  run:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fRA premade html can be found at
 \&\fR\&\f(CW\(ifhttp://www.gnu.org/software/mtools/manual/mtools.html\(is\fR
@@ -249,7 +238,6 @@ To generate an info copy (browsable usin
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
--- a/floppyd_installtest.1	2009-11-04 07:34:01.000000000 +1100
+++ b/floppyd_installtest.1	2010-02-15 14:01:02.000000000 +1100
@@ -18,9 +18,6 @@ documentation, and may not be entirely a
 end of this man page for details.
 .PP
 .SH Description
-.iX "p floppyd_installtest"
-.iX "c X terminal"
-.iX "c remote floppy access"
 .PP
 \&\fR\&\f(CWFloppyd_installtest\fR is used to check for the presence of a running
 floppyd daemon. This is usefull, if you have a small frontend script to
@@ -57,7 +54,6 @@ commands:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .TP
@@ -71,7 +67,6 @@ To generate a html copy,  run:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fRA premade html can be found at
 \&\fR\&\f(CW\(ifhttp://www.gnu.org/software/mtools/manual/mtools.html\(is\fR
@@ -86,7 +81,6 @@ To generate an info copy (browsable usin
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
--- a/mattrib.1	2010-02-15 12:37:35.000000000 +1100
+++ b/mattrib.1	2010-02-15 14:01:02.000000000 +1100
@@ -18,12 +18,6 @@ documentation, and may not be entirely a
 end of this man page for details.
 .PP
 .SH Description
-.iX "p mattrib"
-.iX "c Changing file attributes"
-.iX "c Hidden files"
-.iX "c Read-only files (changing the attribute)"
-.iX "c System files"
-.iX "c Archive bit"
 .PP
 \&\fR\&\f(CWMattrib\fR is used to change MS-DOS file attribute flags. It has the
 following syntax:
@@ -94,7 +88,6 @@ commands:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .TP
@@ -108,7 +101,6 @@ To generate a html copy,  run:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fRA premade html can be found at
 \&\fR\&\f(CW\(ifhttp://www.gnu.org/software/mtools/manual/mtools.html\(is\fR
@@ -123,7 +115,6 @@ To generate an info copy (browsable usin
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
--- a/mbadblocks.1	2009-11-04 07:34:01.000000000 +1100
+++ b/mbadblocks.1	2010-02-15 14:01:02.000000000 +1100
@@ -23,10 +23,6 @@ The \fR\&\f(CWmbadblocks\fR command is u
 its unused bad blocks as bad. It uses the following syntax:
 .PP
 \&\fR\&\f(CWmbadblocks\fR \fIdrive\fR\fR\&\f(CW:\fR
-.iX "p mbadblocks"
-.iX "c Marking blocks as bad"
-.iX "c Bad blocks"
-.iX "c Read errors"
 .PP
 \&\fR\&\f(CWMbadblocks\fR scans an MS-DOS floppy for bad blocks. All unused bad
 blocks are marked as such in the FAT. This is intended to be used right
@@ -59,7 +55,6 @@ commands:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .TP
@@ -73,7 +68,6 @@ To generate a html copy,  run:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fRA premade html can be found at
 \&\fR\&\f(CW\(ifhttp://www.gnu.org/software/mtools/manual/mtools.html\(is\fR
@@ -88,7 +82,6 @@ To generate an info copy (browsable usin
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
--- a/mcat.1	2009-11-04 07:34:01.000000000 +1100
+++ b/mcat.1	2010-02-15 14:01:02.000000000 +1100
@@ -23,10 +23,6 @@ The \fR\&\f(CWmcat\fR command is used to
 to the floppy device. It uses the following syntax:
 .PP
 \&\fR\&\f(CWmcat\fR [\fR\&\f(CW-w\fR] \fIdrive\fR\fR\&\f(CW:\fR
-.iX "p mcat"
-.iX "c Copying an entire disk image"
-.iX "c Disk image"
-.iX "c Floppyd cat"
 .PP
 \&\fR\&\f(CWMcat\fR performs the same task as the unix \fR\&\f(CWcat\fR command. It
 is included into the mtools package, since \fR\&\f(CWcat\fR cannot access
@@ -64,7 +60,6 @@ commands:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .TP
@@ -78,7 +73,6 @@ To generate a html copy,  run:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fRA premade html can be found at
 \&\fR\&\f(CW\(ifhttp://www.gnu.org/software/mtools/manual/mtools.html\(is\fR
@@ -93,7 +87,6 @@ To generate an info copy (browsable usin
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
--- a/mcd.1	2009-11-04 07:34:01.000000000 +1100
+++ b/mcd.1	2010-02-15 14:01:02.000000000 +1100
@@ -18,12 +18,6 @@ documentation, and may not be entirely a
 end of this man page for details.
 .PP
 .SH Description
-.iX "p mcd"
-.iX "c Directory (changing)"
-.iX "c Working directory"
-.iX "c Current working directory (changing the)"
-.iX "c Default directory (changing the)"
-.iX "c Mcwd file"
 .PP
 The \fR\&\f(CWmcd\fR command is used to change the mtools working directory
 on the MS-DOS disk. It uses the following syntax:
@@ -36,7 +30,6 @@ on the MS-DOS disk. It uses the followin
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
@@ -77,7 +70,6 @@ commands:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .TP
@@ -91,7 +83,6 @@ To generate a html copy,  run:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fRA premade html can be found at
 \&\fR\&\f(CW\(ifhttp://www.gnu.org/software/mtools/manual/mtools.html\(is\fR
@@ -106,7 +97,6 @@ To generate an info copy (browsable usin
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
--- a/mclasserase.1	2009-11-04 07:34:02.000000000 +1100
+++ b/mclasserase.1	2010-02-15 14:01:02.000000000 +1100
@@ -18,9 +18,6 @@ documentation, and may not be entirely a
 end of this man page for details.
 .PP
 .SH Description
-.iX "p mclasserase"
-.iX "c Memory Card"
-.iX "c Physically erase"
 .PP
 The \fR\&\f(CWmclasserase\fR command is used to wipe memory cards by
 overwriting it three times: first with \fR\&\f(CW0xff\fR, then with
@@ -35,7 +32,6 @@ syntax:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
@@ -75,7 +71,6 @@ commands:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .TP
@@ -89,7 +84,6 @@ To generate a html copy,  run:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fRA premade html can be found at
 \&\fR\&\f(CW\(ifhttp://www.gnu.org/software/mtools/manual/mtools.html\(is\fR
@@ -104,7 +98,6 @@ To generate an info copy (browsable usin
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
--- a/mcopy.1	2009-11-04 07:34:01.000000000 +1100
+++ b/mcopy.1	2010-02-15 14:01:02.000000000 +1100
@@ -18,13 +18,6 @@ documentation, and may not be entirely a
 end of this man page for details.
 .PP
 .SH Description
-.iX "p mcopy"
-.iX "c Reading MS-DOS files"
-.iX "c Writing MS-DOS files"
-.iX "c Copying MS-DOS files"
-.iX "c Concatenating MS-DOS files"
-.iX "c Text files"
-.iX "c CR/LF conversions"
 .PP
 The \fR\&\f(CWmcopy\fR command is used to copy MS-DOS files to and from
 Unix. It uses the following syntax:
@@ -39,7 +32,6 @@ Unix. It uses the following syntax:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
@@ -117,7 +109,6 @@ mtype a:file1 a:file2 a:file3 | mcopy - 
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
@@ -144,7 +135,6 @@ commands:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .TP
@@ -158,7 +148,6 @@ To generate a html copy,  run:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fRA premade html can be found at
 \&\fR\&\f(CW\(ifhttp://www.gnu.org/software/mtools/manual/mtools.html\(is\fR
@@ -173,7 +162,6 @@ To generate an info copy (browsable usin
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
--- a/mdel.1	2010-02-15 12:37:35.000000000 +1100
+++ b/mdel.1	2010-02-15 14:01:02.000000000 +1100
@@ -20,10 +20,6 @@ documentation, and may not be entirely a
 end of this man page for details.
 .PP
 .SH Description
-.iX "p mdel"
-.iX "c removing MS-DOS files"
-.iX "c erasing MS-DOS files"
-.iX "c deleting MS-DOS files"
 .PP
 The \fR\&\f(CWmdel\fR command is used to delete an MS-DOS file. Its syntax
 is:
@@ -62,7 +58,6 @@ commands:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .TP
@@ -76,7 +71,6 @@ To generate a html copy,  run:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fRA premade html can be found at
 \&\fR\&\f(CW\(ifhttp://www.gnu.org/software/mtools/manual/mtools.html\(is\fR
@@ -91,7 +85,6 @@ To generate an info copy (browsable usin
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
--- a/mdeltree.1	2009-11-04 07:34:01.000000000 +1100
+++ b/mdeltree.1	2010-02-15 14:01:02.000000000 +1100
@@ -18,11 +18,6 @@ documentation, and may not be entirely a
 end of this man page for details.
 .PP
 .SH Description
-.iX "p mdeltree"
-.iX "c removing an MS-DOS directory recursively"
-.iX "c erasing an MS-DOS directory recursively"
-.iX "c deleting an MS-DOS directory recursively"
-.iX "c recursively removing an MS-DOS directory"
 .PP
 The \fR\&\f(CWmdeltree\fR command is used to delete an MS-DOS file. Its syntax
 is:
@@ -61,7 +56,6 @@ commands:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .TP
@@ -75,7 +69,6 @@ To generate a html copy,  run:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fRA premade html can be found at
 \&\fR\&\f(CW\(ifhttp://www.gnu.org/software/mtools/manual/mtools.html\(is\fR
@@ -90,7 +83,6 @@ To generate an info copy (browsable usin
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
--- a/mdir.1	2010-02-15 12:37:35.000000000 +1100
+++ b/mdir.1	2010-02-15 14:01:02.000000000 +1100
@@ -18,10 +18,6 @@ documentation, and may not be entirely a
 end of this man page for details.
 .PP
 .SH Description
-.iX "p mdir"
-.iX "c Read-only files (listing them)"
-.iX "c Listing a directory"
-.iX "c Directory listing"
 .PP
 The \fR\&\f(CWmdir\fR command is used to display an MS-DOS directory. Its
 syntax is:
@@ -82,7 +78,6 @@ commands:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .TP
@@ -96,7 +91,6 @@ To generate a html copy,  run:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fRA premade html can be found at
 \&\fR\&\f(CW\(ifhttp://www.gnu.org/software/mtools/manual/mtools.html\(is\fR
@@ -111,7 +105,6 @@ To generate an info copy (browsable usin
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
--- a/mdu.1	2009-11-04 07:34:02.000000000 +1100
+++ b/mdu.1	2010-02-15 14:01:02.000000000 +1100
@@ -18,11 +18,6 @@ documentation, and may not be entirely a
 end of this man page for details.
 .PP
 .SH Description
-.iX "p mdu"
-.iX "c Space occupied by directories and files"
-.iX "c du"
-.iX "c Listing space occupied by directories and files"
-.iX "c Occupation of space by directories and files"
 .PP
 \&\fR\&\f(CWMdu\fR is used to list the space occupied by a directory, its
 subdirectories and its files. It is similar to the \fR\&\f(CWdu\fR command on
@@ -60,7 +55,6 @@ commands:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .TP
@@ -74,7 +68,6 @@ To generate a html copy,  run:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fRA premade html can be found at
 \&\fR\&\f(CW\(ifhttp://www.gnu.org/software/mtools/manual/mtools.html\(is\fR
@@ -89,7 +82,6 @@ To generate an info copy (browsable usin
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
--- a/mformat.1	2009-11-04 07:34:02.000000000 +1100
+++ b/mformat.1	2010-02-15 14:01:02.000000000 +1100
@@ -18,10 +18,6 @@ documentation, and may not be entirely a
 end of this man page for details.
 .PP
 .SH Description
-.iX "p mformat"
-.iX "c Initializing disks"
-.iX "c Formatting disks"
-.iX "c Filesystem creation"
 .PP
 The \fR\&\f(CWmformat\fR command is used to add an MS-DOS filesystem to a
 low-level formatted diskette. Its syntax is:
@@ -243,7 +239,6 @@ commands:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .TP
@@ -257,7 +252,6 @@ To generate a html copy,  run:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fRA premade html can be found at
 \&\fR\&\f(CW\(ifhttp://www.gnu.org/software/mtools/manual/mtools.html\(is\fR
@@ -272,7 +266,6 @@ To generate an info copy (browsable usin
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
--- a/minfo.1	2009-11-04 07:34:02.000000000 +1100
+++ b/minfo.1	2010-02-15 14:01:02.000000000 +1100
@@ -18,9 +18,6 @@ documentation, and may not be entirely a
 end of this man page for details.
 .PP
 .SH Description
-.iX "p minfo"
-.iX "c mformat parameters"
-.iX "c getting parameters of a Dos fs"
 .PP
 The \fR\&\f(CWminfo\fR command prints the parameters of a Dos filesystem, such
 as number of sectors, heads and cylinders.  It also prints an mformat
@@ -62,7 +59,6 @@ commands:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .TP
@@ -76,7 +72,6 @@ To generate a html copy,  run:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fRA premade html can be found at
 \&\fR\&\f(CW\(ifhttp://www.gnu.org/software/mtools/manual/mtools.html\(is\fR
@@ -91,7 +86,6 @@ To generate an info copy (browsable usin
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
--- a/mkmanifest.1	2009-11-04 07:34:02.000000000 +1100
+++ b/mkmanifest.1	2010-02-15 14:01:02.000000000 +1100
@@ -18,8 +18,6 @@ documentation, and may not be entirely a
 end of this man page for details.
 .PP
 .SH Description
-.iX "p mkmanifest"
-.iX "c packing list"
 .PP
 The \fR\&\f(CWmkmanifest\fR command is used to create a shell script (packing
 list) to restore Unix filenames. Its syntax is:
@@ -54,7 +52,6 @@ You want to copy the following Unix file
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
@@ -74,7 +71,6 @@ converts the names to:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
@@ -87,7 +83,6 @@ mkmanifest very_long_name 2.many.dots il
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fRwould produce the following:
  
@@ -102,7 +97,6 @@ mkmanifest very_long_name 2.many.dots il
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
@@ -142,7 +136,6 @@ commands:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .TP
@@ -156,7 +149,6 @@ To generate a html copy,  run:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fRA premade html can be found at
 \&\fR\&\f(CW\(ifhttp://www.gnu.org/software/mtools/manual/mtools.html\(is\fR
@@ -171,7 +163,6 @@ To generate an info copy (browsable usin
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
--- a/mlabel.1	2009-11-04 07:34:02.000000000 +1100
+++ b/mlabel.1	2010-02-15 14:01:02.000000000 +1100
@@ -18,9 +18,6 @@ documentation, and may not be entirely a
 end of this man page for details.
 .PP
 .SH Description
-.iX "p mlabel"
-.iX "c Labeling a disk"
-.iX "c Disk label"
 .PP
 The \fR\&\f(CWmlabel\fR command adds a volume label to a disk. Its syntax is:
 .ft I
@@ -78,7 +75,6 @@ commands:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .TP
@@ -92,7 +88,6 @@ To generate a html copy,  run:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fRA premade html can be found at
 \&\fR\&\f(CW\(ifhttp://www.gnu.org/software/mtools/manual/mtools.html\(is\fR
@@ -107,7 +102,6 @@ To generate an info copy (browsable usin
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
--- a/mmd.1	2009-11-04 07:34:02.000000000 +1100
+++ b/mmd.1	2010-02-15 14:01:02.000000000 +1100
@@ -18,11 +18,6 @@ documentation, and may not be entirely a
 end of this man page for details.
 .PP
 .SH Description
-.iX "p mmd"
-.iX "c Making a directory"
-.iX "c Creating a directory"
-.iX "c Directory creation"
-.iX "c Subdirectory creation"
 .PP
 The \fR\&\f(CWmmd\fR command is used to make an MS-DOS subdirectory. Its
 syntax is:
@@ -56,7 +51,6 @@ commands:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .TP
@@ -70,7 +64,6 @@ To generate a html copy,  run:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fRA premade html can be found at
 \&\fR\&\f(CW\(ifhttp://www.gnu.org/software/mtools/manual/mtools.html\(is\fR
@@ -85,7 +78,6 @@ To generate an info copy (browsable usin
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
--- a/mmount.1	2009-11-04 07:34:02.000000000 +1100
+++ b/mmount.1	2010-02-15 14:01:02.000000000 +1100
@@ -18,10 +18,6 @@ documentation, and may not be entirely a
 end of this man page for details.
 .PP
 .SH Description
-.iX "p mmount"
-.iX "c Linux enhancements (mmount)"
-.iX "c Mounting a disk"
-.iX "c High capacity formats, mounting"
 .PP
 The \fR\&\f(CWmmount\fR command is used to mount an MS-DOS disk. It is only
 available on Linux, as it is only useful if the OS kernel allows to
@@ -60,7 +56,6 @@ commands:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .TP
@@ -74,7 +69,6 @@ To generate a html copy,  run:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fRA premade html can be found at
 \&\fR\&\f(CW\(ifhttp://www.gnu.org/software/mtools/manual/mtools.html\(is\fR
@@ -89,7 +83,6 @@ To generate an info copy (browsable usin
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
--- a/mmove.1	2009-11-04 07:34:02.000000000 +1100
+++ b/mmove.1	2010-02-15 14:01:02.000000000 +1100
@@ -18,9 +18,6 @@ documentation, and may not be entirely a
 end of this man page for details.
 .PP
 .SH Description
-.iX "p mmove"
-.iX "c Moving files (mmove)"
-.iX "c Renaming files (mmove)"
 .PP
 The \fR\&\f(CWmmove\fR command is used to moves or renames an existing MS-DOS
 file or subdirectory.
@@ -62,7 +59,6 @@ commands:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .TP
@@ -76,7 +72,6 @@ To generate a html copy,  run:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fRA premade html can be found at
 \&\fR\&\f(CW\(ifhttp://www.gnu.org/software/mtools/manual/mtools.html\(is\fR
@@ -91,7 +86,6 @@ To generate an info copy (browsable usin
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
--- a/mpartition.1	2010-02-15 12:37:35.000000000 +1100
+++ b/mpartition.1	2010-02-15 14:01:02.000000000 +1100
@@ -18,10 +18,6 @@ documentation, and may not be entirely a
 end of this man page for details.
 .PP
 .SH Description
-.iX "p mpartition"
-.iX "c partitions (creating)"
-.iX "c Zip disks (partitioning them)"
-.iX "c Jaz disks (partitioning them)"
 .PP
 The \fR\&\f(CWmpartition\fR command is used to create MS-DOS filesystems as
 partitions.  This is intended to be used on non-Linux systems,
@@ -148,7 +144,6 @@ commands:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .TP
@@ -162,7 +157,6 @@ To generate a html copy,  run:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fRA premade html can be found at
 \&\fR\&\f(CW\(ifhttp://www.gnu.org/software/mtools/manual/mtools.html\(is\fR
@@ -177,7 +171,6 @@ To generate an info copy (browsable usin
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
--- a/mrd.1	2009-11-04 07:34:02.000000000 +1100
+++ b/mrd.1	2010-02-15 14:01:02.000000000 +1100
@@ -18,12 +18,6 @@ documentation, and may not be entirely a
 end of this man page for details.
 .PP
 .SH Description
-.iX "p mrd"
-.iX "c Removing a directory"
-.iX "c Erasing a directory"
-.iX "c Deleting a directory"
-.iX "c Directory removing"
-.iX "c Subdirectory removing"
 .PP
 The \fR\&\f(CWmrd\fR command is used to remove an MS-DOS subdirectory. Its
 syntax is:
@@ -61,7 +55,6 @@ commands:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .TP
@@ -75,7 +68,6 @@ To generate a html copy,  run:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fRA premade html can be found at
 \&\fR\&\f(CW\(ifhttp://www.gnu.org/software/mtools/manual/mtools.html\(is\fR
@@ -90,7 +82,6 @@ To generate an info copy (browsable usin
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
--- a/mren.1	2009-11-04 07:34:02.000000000 +1100
+++ b/mren.1	2010-02-15 14:01:02.000000000 +1100
@@ -18,9 +18,6 @@ documentation, and may not be entirely a
 end of this man page for details.
 .PP
 .SH Description
-.iX "p mren"
-.iX "c Renaming files (mren)"
-.iX "c Moving files (mren)"
 .PP
 The \fR\&\f(CWmren\fR command is used to rename or move an existing MS-DOS
 file or subdirectory. Its syntax is:
@@ -68,7 +65,6 @@ commands:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .TP
@@ -82,7 +78,6 @@ To generate a html copy,  run:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fRA premade html can be found at
 \&\fR\&\f(CW\(ifhttp://www.gnu.org/software/mtools/manual/mtools.html\(is\fR
@@ -97,7 +92,6 @@ To generate an info copy (browsable usin
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
--- a/mshowfat.1	2009-11-04 07:34:02.000000000 +1100
+++ b/mshowfat.1	2010-02-15 14:01:02.000000000 +1100
@@ -18,9 +18,6 @@ documentation, and may not be entirely a
 end of this man page for details.
 .PP
 .SH Description
-.iX "p mshowfat"
-.iX "c Clusters of a file"
-.iX "c Fat"
 .PP
 The \fR\&\f(CWmshowfat\fR command is used to display the FAT entries for a
 file.  Syntax:
@@ -55,7 +52,6 @@ commands:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .TP
@@ -69,7 +65,6 @@ To generate a html copy,  run:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fRA premade html can be found at
 \&\fR\&\f(CW\(ifhttp://www.gnu.org/software/mtools/manual/mtools.html\(is\fR
@@ -84,7 +79,6 @@ To generate an info copy (browsable usin
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
--- a/mtools.1	2010-02-15 13:38:09.000000000 +1100
+++ b/mtools.1	2010-02-15 14:01:02.000000000 +1100
@@ -30,11 +30,6 @@ without any preliminary mounting or init
 change floppies too without unmounting and mounting.
 .PP
 .SH Where\ to\ get\ mtools
-.iX "c bugs"
-.iX "c ALPHA patches"
-.iX "c patches"
-.iX "c diffs"
-.iX "c mailing list"
 .PP
 Mtools can be found at the following places (and their mirrors):
  
@@ -48,7 +43,6 @@ ftp://ibiblio.unc.edu/pub/Linux/utils/di
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
@@ -64,7 +58,6 @@ ftp://www.tux.org/pub/knaff/mtools
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
@@ -86,8 +79,6 @@ list is archived at http://lists.gnu.org
 .SH Common\ features\ of\ all\ mtools\ commands
 .PP
 .SS Options\ and\ filenames
-.iX "c Filenames"
-.iX "c Options"
 MS-DOS filenames are composed of a drive letter followed by a colon, a
 subdirectory, and a filename. Only the filename part is mandatory, the
 drive letter and the subdirectory are optional. Filenames without a
@@ -142,7 +133,6 @@ Example:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
@@ -161,7 +151,6 @@ Example:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
@@ -169,11 +158,6 @@ This looks for the image at the offset o
 at its beginning.
 .PP
 .SS Current\ working\ directory
-.iX "p mcd (introduction)"
-.iX "c Directory"
-.iX "c Working directory"
-.iX "c Current working directory"
-.iX "c Default directory"
 .PP
 The \fR\&\f(CWmcd\fR command (\(ifmcd\(is) is used to establish the device and
 the current working directory (relative to the MS-DOS filesystem),
@@ -182,11 +166,6 @@ MS-DOS, there is only one working direct
 per drive.
 .PP
 .SS VFAT-style\ long\ file\ names
-.iX "c Long file name"
-.iX "c Windows 95-style file names"
-.iX "c VFAT-style file names"
-.iX "c Primary file name (long names)"
-.iX "c Secondary file name (long names)"
 .PP
 This version of mtools supports VFAT style long filenames. If a Unix
 filename is too long to fit in a short DOS name, it is stored as a
@@ -209,7 +188,6 @@ hot+cold        HOT_CO~1        illegal 
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
@@ -243,7 +221,6 @@ the \fIprimary\fR name, and the derived 
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR Mtools creates a VFAT entry for Reallylongname, and uses REALLYLO as
 a short name. Reallylongname is the primary name, and REALLYLO is the
@@ -256,7 +233,6 @@ secondary name.
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR Motd fits into the DOS filename limits. Mtools doesn't need to
 derivate another name. Motd is the primary name, and there is no
@@ -281,7 +257,6 @@ ab:c            ab_c-1          illegal 
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
@@ -295,11 +270,6 @@ Illegal characters are replaces by under
 A \fR\&\f(CW-\fR\fIn\fR number is generated,
 .PP
 .SS Name\ clashes
-.iX "c Name clashes"
-.iX "c Duplicate file names"
-.iX "c Overwriting files"
-.iX "c Primary file name (name clashes)"
-.iX "c Secondary file name (name clashes)"
 .PP
 When writing a file to disk, its long name or short name may collide
 with an already existing file or directory. This may happen for all
@@ -375,7 +345,6 @@ If a name clash occurs in a Unix directo
 to overwrite the file, or to skip it.
 .PP
 .SS Case\ sensitivity\ of\ the\ VFAT\ filesystem
-.iX "c Case sensitivity"
 .PP
 The VFAT filesystem is able to remember the case of the
 filenames. However, filenames which differ only in case are not allowed
@@ -396,12 +365,6 @@ from the behavior of the old version of 
 lower case Unix filenames.
 .PP
 .SS high\ capacity\ formats
-.iX "c Special formats"
-.iX "c High capacity formats"
-.iX "c Odd formats"
-.iX "c Weird formats"
-.iX "c Formats, high capacity"
-.iX "c Linux enhancements (High Capacity Formats)"
 .PP
 Mtools supports a number of formats which allow to store more data on
 disk as usual. Due to different operating system abilities, these
@@ -420,7 +383,6 @@ specific tool. For Linux, suitable flopp
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
@@ -429,10 +391,6 @@ See the manpages included in that packag
 \&\fR\&\f(CWxdfcopy\fR to format XDF.
 .PP
 .SS \ \ More\ sectors
-.iX "c fdformat"
-.iX "c vgacopy"
-.iX "c DMF disks"
-.iX "c Windows 95 (DMF disks)"
 .PP
 The oldest method of fitting more data on a disk is to use more sectors
 and more cylinders. Although the standard format uses 80 cylinders and
@@ -451,7 +409,6 @@ years before! Mtools supports these form
 the DELL Unix PC.
 .PP
 .SS \ \ Bigger\ sectors
-.iX "c bigger sectors"
 By using bigger sectors it is possible to go beyond the capacity which
 can be obtained by the standard 512-byte sectors. This is because of the
 sector header. The sector header has the same size, regardless of how
@@ -465,7 +422,6 @@ This method allows to store up to 1992K 
 Mtools supports these formats only on Linux.
 .PP
 .SS \ \ 2m
-.iX "c 2m"
 .PP
 The 2m format was originally invented by Ciriaco Garcia de Celis. It
 also uses bigger sectors than usual in order to fit more data on the
@@ -488,8 +444,6 @@ to store anything else than a DOS fs\fR.
 Mtools supports these formats only on Linux.
 .PP
 .SS \ \ XDF
-.iX "c XDF disks"
-.iX "c OS/2 (XDF disks)"
 .PP
 XDF is a high capacity format used by OS/2. It can hold 1840 K per
 disk. That's lower than the best 2m formats, but its main advantage is
--- a/mtoolstest.1	2009-11-04 07:34:02.000000000 +1100
+++ b/mtoolstest.1	2010-02-15 14:01:02.000000000 +1100
@@ -18,10 +18,6 @@ documentation, and may not be entirely a
 end of this man page for details.
 .PP
 .SH Description
-.iX "p mtoolstest"
-.iX "c Testing configuration file for correctness"
-.iX "c Checking configuration file"
-.iX "c Verifying configuration file"
 .PP
 The \fR\&\f(CWmtoolstest\fR command is used to tests the mtools configuration
 files. To invoke it, just type \fR\&\f(CWmtoolstest\fR without any arguments.
@@ -54,7 +50,6 @@ commands:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .TP
@@ -68,7 +63,6 @@ To generate a html copy,  run:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fRA premade html can be found at
 \&\fR\&\f(CW\(ifhttp://www.gnu.org/software/mtools/manual/mtools.html\(is\fR
@@ -83,7 +77,6 @@ To generate an info copy (browsable usin
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
--- a/mtype.1	2009-11-04 07:34:02.000000000 +1100
+++ b/mtype.1	2010-02-15 14:01:02.000000000 +1100
@@ -74,7 +74,6 @@ commands:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .TP
@@ -88,7 +87,6 @@ To generate a html copy,  run:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fRA premade html can be found at
 \&\fR\&\f(CW\(ifhttp://www.gnu.org/software/mtools/manual/mtools.html\(is\fR
@@ -103,7 +101,6 @@ To generate an info copy (browsable usin
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
--- a/mzip.1	2009-11-04 07:34:02.000000000 +1100
+++ b/mzip.1	2010-02-15 14:01:02.000000000 +1100
@@ -18,15 +18,6 @@ documentation, and may not be entirely a
 end of this man page for details.
 .PP
 .SH Description
-.iX "c Zip disk (utilities)"
-.iX "c Jaz disk (utilities)"
-.iX "c Ejecting a Zip/Jaz disk"
-.iX "c Write protecting a Zip/Jaz disk"
-.iX "p mzip"
-.iX "c ZipTools disk"
-.iX "c Tools disk (Zip and Jaz drives)"
-.iX "c APlaceForYourStuff"
-.iX "c password protected Zip disks"
 .PP
 The \fR\&\f(CWmzip\fR command is used to issue ZIP disk specific commands on
 Linux, Solaris or HPUX. Its syntax is:
@@ -116,7 +107,6 @@ commands:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .TP
@@ -130,7 +120,6 @@ To generate a html copy,  run:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fRA premade html can be found at
 \&\fR\&\f(CW\(ifhttp://www.gnu.org/software/mtools/manual/mtools.html\(is\fR
@@ -145,7 +134,6 @@ To generate an info copy (browsable usin
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
--- a/mtools.5	2010-02-15 12:12:13.000000000 +1100
+++ b/mtools.5	2010-02-15 14:01:02.000000000 +1100
@@ -14,8 +14,6 @@ mtools.conf - mtools configuration files
 .tr \(pd"
 
 .ds St Mtools\ 4.0.12
-.oh '\\*(St''%'
-.eh '%''\\*(St'
 .PP
 .SH Description
 .PP
@@ -31,9 +29,6 @@ files describe the following items:
 .PP
 .SS Location\ of\ the\ configuration\ files
 .PP
-.iX "c Configuration file name"
-.iX "c Name of configuration files"
-.iX "c Location of configuration files"
 .PP
 \&\fR\&\f(CW\(if/etc/mtools.conf\(is\fR is the system-wide configuration file,
 and \fR\&\f(CW\(if~/.mtoolsrc\(is\fR is the user's private configuration file.
@@ -42,8 +37,6 @@ On some systems, the system-wide configu
 \&\fR\&\f(CW\(if/etc/default/mtools.conf\(is\fR instead.
 .PP
 .SS \ \ General\ configuration\ file\ syntax
-.iX "c Syntax of the configuration file"
-.iX "c Configuration file syntax"
 .PP
 The configuration files is made up of sections. Each section starts
 with a keyword identifying the section followed by a colon.
@@ -65,9 +58,6 @@ configuration file is case insensitive, 
 quotes (such as filenames).
 .PP
 .SS Default\ values
-.iX "c Default values"
-.iX "c Default configuration"
-.iX "c Configuration file"
 For most platforms, mtools contains reasonable compiled-in defaults for
 physical floppy drives.  Thus, you usually don't need to bother with the
 configuration file, if all you want to do with mtools is to access your
@@ -76,14 +66,6 @@ you also want to use mtools to access yo
 dosemu image files.
 .PP
 .SS Global\ variables
-.iX "c Global configuration variables"
-.iX "c Drive independent configuration variables"
-.iX "c Environmental variables"
-.iX "v MTOOLS_SKIP_CHECK"
-.iX "v MTOOLS_FAT_COMPATIBILITY"
-.iX "v MTOOLS_LOWER_CASE"
-.iX "v MTOOLS_NO_VFAT"
-.iX "c FreeDos"
 .PP
 Global flags may be set to 1 or to 0.
 .PP
@@ -134,7 +116,6 @@ mtools to skip the sanity checks:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
@@ -147,7 +128,6 @@ Global variables may also be set via the
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
@@ -157,13 +137,8 @@ Global string variables may be set to an
 The format used for printing dates of files.  By default, is dd-mm-yyyy.
 .PP
 .SS Per\ drive\ flags\ and\ variables
-.iX "c Drive description"
-.iX "c Drive configuration"
 .PP
 .SS \ \ General\ information
-.iX "c Drive description, example"
-.iX "c Drive configuration, example"
-.iX "v drive"
 .PP
 Per drive flags and values may be described in a drive section. A
 drive section starts with
@@ -181,34 +156,19 @@ This is a sample drive description:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
 .SS \ \ Location\ information
-.iX "c Hdimage"
 .PP
 For each drive, you need to describe where its data is physically
 stored (imag file, physical device, partition, offset).
 .TP
 \&\fR\&\f(CWfile\fR\ 
-.iX "c Image file"
-.iX "c Name of device node"
-.iX "c File name of device node"
-.iX "v file"
 The name of the file or device holding the disk image. This is
 mandatory. The file name should be enclosed in quotes.
 .TP
 \&\fR\&\f(CWpartition\fR\ 
-.iX "c Dosemu hard disk image"
-.iX "c Zip disks (partitions)"
-.iX "c Jaz disks (partitions)"
-.iX "c Syquest disks"
-.iX "c Magneto-optical disks"
-.iX "c OS/2 (layout of removable media)"
-.iX "c Windows NT (layout of removable media)"
-.iX "c Removable media"
-.iX "c Partitioned image file"
 Tells mtools to treat the drive as a partitioned device, and to use the
 given partition. Only primary partitions are accessible using this
 method, and they are numbered from 1 to 4. For logical partitions, use
@@ -222,21 +182,12 @@ hdimages. It is not recommended for hard
 to partitions is available through mounting.
 .TP
 \&\fR\&\f(CWoffset\fR\ 
-.iX "c Ram disk"
-.iX "c Atari Ram disk"
 Describes where in the file the MS-DOS filesystem starts. This is useful
 for logical partitions in DOSEMU hdimages, and for ATARI ram disks. By
 default, this is zero, meaning that the filesystem starts right at the
 beginning of the device or file.
 .PP
 .SS \ \ Disk\ Geometry\ Configuration
-.iX "c Disk Geometry"
-.iX "c Configuration of disk geometry"
-.iX "c Description of disk geometry"
-.iX "c Format of disk"
-.iX "c High density disk"
-.iX "c Low density disk"
-.iX "p mformat (geometry used for)"
 .PP
 Geometry information describes the physical characteristics about the
 disk. Its has three purposes:
@@ -292,17 +243,13 @@ The following geometry related variables
 \&\fR\&\f(CWcylinders\fR\ 
 .TQ
 \&\fR\&\f(CWtracks\fR
-.iX "v cylinders"
-.iX "v tracks"
 The number of cylinders. (\fR\&\f(CWcylinders\fR is the preferred form,
 \&\fR\&\f(CWtracks\fR is considered obsolete)
 .TP
 \&\fR\&\f(CWheads\fR\ 
-.iX "v heads"
 The number of heads (sides).
 .TP
 \&\fR\&\f(CWsectors\fR\ 
-.iX "v sectors"
 The number of sectors per track.
 .PP
 Example: the following drive section describes a 1.44M drive:
@@ -319,7 +266,6 @@ Example: the following drive section des
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
@@ -347,12 +293,6 @@ describes a 320k disk and is equivalent 
 \&\fR\&\f(CWfat_bits=12 cylinders=40 heads=2 sectors=8\fR
 .PP
 .SS \ \ Open\ Flags
-.iX "v sync"
-.iX "v nodelay"
-.iX "v exclusive"
-.iX "c open flags"
-.iX "c synchronous writing"
-.iX "c exclusive access to a drive"
 .PP
 Moreover, the following flags are available:
 .TP
@@ -375,7 +315,6 @@ to their type, these variables can be se
 an integer (all others)
 .TP
 \&\fR\&\f(CWfat_bits\fR\ 
-.iX "v fat_bits"
 The number of FAT bits. This may be 12 or 16. This is very rarely
 needed, as it can almost always be deduced from information in the
 boot sector. On the contrary, describing the number of fat bits may
@@ -393,17 +332,12 @@ for all drives by using the global \fR\&
 version 4.0.0
 .TP
 \&\fR\&\f(CWprecmd\fR\ 
-.iX "c Solaris (volcheck)"
-.iX "c Executing commands before opening the device"
 On some variants of Solaris, it is necessary to call 'volcheck -v'
 before opening a floppy device, in order for the system to notice that
 there is indeed a disk in the drive. \fR\&\f(CWprecmd="volcheck -v"\fR in the
 drive clause establishes the desired behavior.
 .TP
 \&\fR\&\f(CWblocksize\fR\ 
-.iX "c raw device"
-.iX "c character devices"
-.iX "c blocksize"
 This parameter represents a default block size to be always used on this
 device.  All I/O is done with multiples of this block size,
 independantly of the sector size registered in the filesystem's boot
@@ -421,22 +355,12 @@ ommitted, it is enabled.  For example, \
 \&\fR\&\f(CWscsi=1\fR
 .TP
 \&\fR\&\f(CWnolock\fR\ 
-.iX "c disable locking"
-.iX "c locking (disabling it)"
-.iX "c plain floppy: device xxx busy"
 Instruct mtools to not use locking on this drive.  This is needed on
 systems with buggy locking semantics.  However, enabling this makes
 operation less safe in cases where several users may access the same
 drive at the same time.
 .TP
 \&\fR\&\f(CWscsi\fR\ 
-.iX "c setuid installation (needed for raw SCSI I/O)"
-.iX "c Solaris (Raw access to SCSI devices such as Zip & Jaz)"
-.iX "c SunOS (Raw access to SCSI devices such as Zip & Jaz)"
-.iX "c Zip disks (raw Scsi access)"
-.iX "c Jaz disks (raw Scsi access)"
-.iX "c Syquests (raw Scsi access)"
-.iX "c SCSI devices"
 When set to 1, this option tells mtools to use raw SCSI I/O instead of
 the standard read/write calls to access the device. Currently, this is
 supported on HP/UX, Solaris and SunOs.  This is needed because on some
@@ -460,8 +384,6 @@ drives described in a system-wide config
 \&\fR\&\f(CW\(if~/.mtoolsrc\(is\fR or \fR\&\f(CW\(if$MTOOLSRC\(is\fR.  
 .TP
 \&\fR\&\f(CWprivileged\fR\ 
-.iX "c setuid installation"
-.iX "c setgid installation"
 When set to 1, this instructs mtools to use its set-uid and set-gid
 privileges for opening the given drive.  This option is only valid for
 drives described in the system-wide configuration files (such as
@@ -478,8 +400,6 @@ these options, mtools works perfectly we
 setuid root.
 .TP
 \&\fR\&\f(CWvold\fR\ 
-.iX "c Solaris (vold)"
-.iX "c Vold (mediamgr)"
 .IP
 Instructs mtools to interpret the device name as a vold identifier
 rather than as a filename.  The vold identifier is translated into a
@@ -489,25 +409,19 @@ flag is only available if you configured
 \&\fR\&\f(CW--enable-new-vold\fR option before compilation.
 .TP
 \&\fR\&\f(CWswap\fR\ 
-.iX "c Atari"
-.iX "c Wordswapped"
 .IP
 Consider the media as a word-swapped Atari disk.
 .TP
 \&\fR\&\f(CWuse_xdf\fR\ 
-.iX "c XDF disks (how to configure)"
-.iX "v use_xdf"
 If this is set to a non-zero value, mtools also tries to access this
 disk as an XDF disk. XDF is a high capacity format used by OS/2. This
 is off by default. See section XDF, for more details.
 .TP
 \&\fR\&\f(CWmformat_only\fR\ 
-.iX "v mformat_only"
 Tells mtools to use the geometry for this drive only for mformatting and 
 not for filtering.
 .TP
 \&\fR\&\f(CWfilter\fR\ 
-.iX "v filter"
 Tells mtools to use the geometry for this drive both for mformatting and 
 filtering.
 .TP
@@ -541,7 +455,6 @@ Example:
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
@@ -562,7 +475,6 @@ physical drives through one drive letter
 .fi
 .in -0.3i
 .ft R
-.lp
  
 \&\fR
 .PP
@@ -578,11 +490,6 @@ end of the list (i.e. it will be tried l
 the start of the list.
 .PP
 .SS Location\ of\ configuration\ files\ and\ parsing\ order
-.iX "c Parsing order"
-.iX "c Configuration file parsing order"
-.iX "c Configuration file name (parsing order)"
-.iX "c Name of configuration files (parsing order)"
-.iX "c Location of configuration files (parsing order)"
 .PP
 The configuration files are parsed in the following order:
 .TP
@@ -617,9 +524,6 @@ earlier file, you need to use either the
 keyword.
 .PP
 .SS Backwards\ compatibility\ with\ old\ configuration\ file\ syntax
-.iX "c Backwards compatibility"
-.iX "c Old configuration file syntax"
-.iX "c Configuration file, old syntax"
 .PP
 The syntax described herein is new for version \fR\&\f(CWmtools-3.0\fR. The
 old line-oriented syntax is still supported. Each line beginning with a
